### PR TITLE
Added draggable rubric container

### PR DESCRIPTION
**Pull request by Nokondi**
Added react-draggable library to allow for draggable ai rubric.

## Links
Jira: [AITT-468](https://codedotorg.atlassian.net/browse/AITT-468)
[react-draggable @ npmjs](https://www.npmjs.com/package/react-draggable)

https://github.com/code-dot-org/code-dot-org/assets/5552007/1d290a9d-9336-4398-a66a-16bb94eb0935

### CODE REVIEW

**Review by cearachew — COMMENTED** [review]
Looks great!!! Does this need any unit tests/ have you tried it out with the sticky rubric on a reload (e.g. from selecting a student from the dropdown)?

**Comment by tess323** [comment]
Really great work here!  Great find with this library!

File: apps/test/unit/templates/rubrics/RubricContainerTest.jsx [hunk 1]
```diff
@@ -578,4 +580,41 @@ describe('RubricContainer', () => {
...
+
+    // simulate dragging
+    fireEvent.mouseDown(element, {clientX: 0, clientY: 0});
+    fireEvent.mouseMove(element, {clientX: 100, clientY: 100});
+    fireEvent.mouseUp(element);
+
+    const newPosition = element.style.transform;
+
+    expect(newPosition).to.not.equal(initialPosition);
+  });
```
Comment (cearachew) [inline apps/test/unit/templates/rubrics/RubricContainerTest.jsx:630]: yessssss great test

**Review by cearachew — APPROVED** [review]
Looks great!! Thanks for adding the test!

**Pull request merged**